tf2_ros/include/tf2_ros/buffer.h

```diff
@@ -112,21 +112,22 @@ namespace tf2_ros
      * \return True if the transform is possible, false otherwise 
      */
     virtual bool
-      canTransform(const std::string& target_frame, const ros::Time& target_time,
-                   const std::string& source_frame, const ros::Time& source_time,
-                   const std::string& fixed_frame, const ros::Duration timeout, std::string* errstr = NULL) const;
+      canTransform(const std::string& target_frame, const tf2::TimePoint& target_time,
+                   const std::string& source_frame, const tf2::TimePoint& source_time,
+                   const std::string& fixed_frame, const tf2::TempDuration timeout, std::string* errstr = NULL) const;
 
 
     
     
   private:
-    bool getFrames(tf2_msgs::FrameGraph::Request& req, tf2_msgs::FrameGraph::Response& res) ;
+    bool getFrames(tf2_msgs::srv::FrameGraph::Request& req, tf2_msgs::srv::FrameGraph::Response& res) ;
 
 
     // conditionally error if dedicated_thread unset.
     bool checkAndErrorDedicatedThreadPresent(std::string* errstr) const;
 
-    ros::ServiceServer frames_server_;
+//TODO(renable framegraph service)
```

Comment:
nitpik: `// TODO(tfoote) re-enable frame graph service`, also what is a frame graph service (for my own edification)?